cypress-io/cypress
# refactor: improve angular mount log and override build options for dev

### User facing changelog
na

### Additional details
Pairing with @jordanpowell88 on Angular stuff and came across some slight improvements to the mount and dev-server. We now bind the mount response to the `Cypress.log` of the mount command, and we made a few build overrides based on the [@angular/cli](https://github.com/angular/angular-cli/blob/96d39f8f3a6fcb56aa30cbf61dcd96dfcac55621/packages/angular_devkit/build_angular/src/builders/dev-server/index.ts#L132) dev-server overrides.

### Steps to test
You can test the mount by running `yarn workspace @cypress/webpack-dev-server` and clicking on the `angular.cy.ts` test. Make sure to run `yarn` to build everything first.

### How has the user experience changed?
Video shows the output of clicking on the `mount` command in the command log and the network requests shows the removal of the bundle hashing e.g. `spec-0.abfj8083urn.js -> spec-0.js`

https://user-images.githubusercontent.com/25158820/181293578-66953b07-5b0a-4ff8-9f64-2c058da1cb71.mov



### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
